Assets/Scripts/Dialogue/ActivateTextAtLine.cs

```diff
@@ -34,7 +34,7 @@ public class ActivateTextAtLine : MonoBehaviour {
 
     void OnTriggerEnter2D(Collider2D other)
     {
-		if (other.CompareTag ("Player") || other.CompareTag ("Doll")) 
+		if (other.CompareTag ("Player")) 
```

Comment:
yes, but this is for activation of dialogue, so i don't think dolls without souls should be able to do activate texts